src/app/kvm/components/KVMHeaderForms/AddVirsh/AddVirshFields/AddVirshFields.test.tsx

```diff
@@ -71,26 +64,17 @@ describe("AddVirshFields", () => {
       }),
     ];
     state.general.powerTypes.data = powerTypes;
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[
-            { pathname: "/machines/chassis/add", key: "testKey" },
-          ]}
-        >
-          <CompatRouter>
-            <AddVirsh clearSidePanelContent={jest.fn()} />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
-    );
 
-    expect(wrapper.find("Input[name='power_parameters.field1']").exists()).toBe(
-      true
-    );
-    expect(wrapper.find("Input[name='power_parameters.field2']").exists()).toBe(
-      false
-    );
+    renderWithBrowserRouter(<AddVirsh clearSidePanelContent={jest.fn()} />, {
+      state,
+      route: "/machines/chassis/add",
+    });
+
+    expect(
+      screen.getByRole("textbox", { name: /test-powerfield-label-1/i })
+    ).toBeInTheDocument();
```

Comment:
You should be able to set a label directly for each power field in the state (line 60), this way you can check it directly and the test should be more solid